api/apps/geoprocessing/src/modules/surface-cost/adapters/extract-pu-cost.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { Test } from '@nestjs/testing';
+import { PromiseType } from 'utility-types';
+import {
+  getGeoJson,
+  getGeoJsonWithMissingCost,
+  getGeometryMultiPolygon,
+} from '../application/__mocks__/geojson';
+import { ExtractPuCost } from './extract-pu-cost';
+
+let sut: ExtractPuCost;
+let fixtures: PromiseType<ReturnType<typeof getFixtures>>;
+
+beforeEach(async () => {
+  fixtures = await getFixtures();
+  sut = fixtures.getService();
+});
+
+describe(`when features miss cost`, () => {
+  it(`throws exception`, () => {
+    expect(() => sut.extract(fixtures.geoFeaturesWithoutCost())).toThrow(
+      /missing cost/,
+    );
+  });
+});
+
+describe(`when given GeoJson isn't a FeatureCollection`, () => {
+  it(`throws exception`, () => {
+    expect(() => sut.extract(fixtures.simpleGeometry())).toThrow(
+      /is supported/,
```

Comment:
super minor - we'll see the test passing, but asserting that an exception will be raised containing the text `is supported` seems slightly misleading (why should an error be thrown if *something* "is supported"?): in this specific case, maybe matching case-insensitively the full text of the error (`/Only FeatureCollection is supported/i`) should make things clearer for whoever skims through these tests in the future.

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/219/commits/573622f92a11fdd126982184dc16a4ab3e03ecb3